test/bruno/1 - REST API/10 - Branding/23 - Render Preview CSS.bru

```diff
@@ -0,0 +1,24 @@
+meta {
+  name: Render Preview CSS Success
```

Comment:
The name says success, but the tests are 200 or 404. What is this test expecting?